app/scripts/models/user.js

```diff
@@ -532,12 +532,25 @@ define(function (require, exports, module) {
     },
 
     /**
+     * Fetch the clients for the given account, populated the passed in
+     * Clients collection.
+     *
+     * @param {Object} account - account for which device list is requested
+     * @param {Object} apps - Clients collection used to store list.
+     * @returns {Promise} resolves when the action completes
+     */
+    fetchAccountApps: function (account, apps) {
+      return this._oAuthClient.fetchClients(account.get('accessToken'), apps)
```

Comment:
Hmm.. These will be the first functions within user.js that make use of the OAuthClient directly. Can you push these functions down into account.js so we limit direct access to the `accessToken`? [account.js can re-provision an accessToken](https://github.com/mozilla/fxa-content-server/blob/58af33b0eeb1fa2a47ac7fdc54e0ed12b9ad10e0/app/scripts/models/account.js#L849) for the profile server if needed, I imagine that could be used here too.

Also, it looks like `accessToken` has so far been used for interacting with the auth server, now the accessToken will also be used to interact with the OAuth server, is that expected?